doc/conf.py

```diff
@@ -148,22 +148,23 @@ def inner(fn, *iargs, **ikwargs):  # pylint: disable=unused-argument
     "MySQLdb",
     "MySQLdb.cursors",
     "OpenSSL",
+    "ansible",
     "avahi",
     "boto.regioninfo",
+    "concurrent",
```

Comment:
Looks like this is causing the docs failure

Reply:
this is fixed on master I think.  This probably needs to be rebased and will then be fine.

Reply:
Doesn't look like it. On the other hand, it looks like I added `ansible` and `concurrent` to fix the doc build on my end, but apparently broke it in master :)

Reply:
yup, that sounds correct after looking at it closer.